### PR TITLE
Update vanilla Akka gRPC to not use TLS in examples

### PR DESCRIPTION
**Pull request by chbatey**
Our examples just got a lot smaller

### CODE REVIEW

**Comment by chbatey** [comment]
We need to either revert the changes for Java or release Akka HTTP due to: https://github.com/akka/akka-http/pull/2149

So don't merge for now

File: docs/src/main/paradox/overview.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ This library is in preview mode: basic functionality is in place, but API's and
 build system plugins are still expected to be improved.
 
 The API on both sides (Client and Server) is a simple Akka Streams-based one.
-We plan to also provide a 'power user' API for each of these ([#191](https://github.com/akka/akka-grpc/issues/191), [#179](https://github.com/akka/akka-grpc/issues/179)).
+The client has a 'power user' API and the planned for the server in [#179](https://github.com/akka/akka-grpc/issues/179)).
```
Comment (johanandren) [inline docs/src/main/paradox/overview.md]: "and the planned for the server in"?

**Comment by ktoso** [comment]
I vote for releasing akka-http, was a while since we did anyway :)

File: plugin-tester-java/src/main/java/example/myapp/CombinedServer.java [hunk 2]
```diff
@@ -63,54 +60,11 @@ public static void main(String[] args) throws Exception {
 
       Http.get(sys).bindAndHandleAsync(
           serviceHandlers,
-          ConnectWithHttps.toHostHttps("127.0.0.1", 8080).withCustomHttpsContext(serverHttpContext()),
+          ConnectHttp.toHost("127.0.0.1", 8080, UseHttp2.always()),
```
Comment (johanandren) [inline plugin-tester-java/src/main/java/example/myapp/CombinedServer.java]: Does this really work? https://github.com/akka/akka-http/issues/2110

Comment (chbatey) [reply plugin-tester-java/src/main/java/example/myapp/CombinedServer.java]: No, see comment above. Needs an akka http release

**Pull request merged**